upup/pkg/fi/cloudup/template_functions.go

```diff
@@ -526,3 +527,23 @@ func (tf *TemplateFunctions) OpenStackCCMTag() string {
 	}
 	return tag
 }
+
+func (tf *TemplateFunctions) CasTag() (tag string) {
+	tag = "latest"
+	parsed, err := util.ParseKubernetesVersion(tf.Cluster.Spec.KubernetesVersion)
+	if err == nil {
+		switch parsed.Minor {
+		case 19:
+			tag = "v1.19.0"
+		case 18:
+			tag = "v1.18.2"
+		case 17:
+			tag = "v1.17.3"
+		case 16:
+			tag = "v1.16.6"
+		case 15:
+			tag = "v1.15.7"
```

Comment:
I think I'd prefer an error for the default case. Otherwise we might forget to update it for a new Kubernetes minor version.

Reply:
Please respond to this review comment.

Reply:
So this is why I had the validation logic instead. I just felt it nicer should we ever forget to add support here.